package.json

```diff
@@ -32,21 +31,21 @@
     "ember-cli-deploy-frost-pack": "git://github.com/ciena-frost/ember-cli-deploy-frost-pack.git#e98e53709a42604815348f10bc8823c3c425b68e",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mirage": "~0.1.11",
     "ember-cli-mocha": "^0.10.1",
-    "ember-cli-notifications": "^3.2.0",
+    "ember-cli-notifications": "^4.0.6",
     "ember-cli-uglify": "^1.2.0",
+    "ember-code-snippet": "1.4.0",
     "ember-computed-decorators": "0.2.2",
     "ember-concurrency": "0.7.8",
-    "ember-data": "^2.4.0",
+    "ember-data": "^2.8.0",
```

Comment:
Remove ember data since I don't think it's used.  Otherwise keep in sync with ember and ember cli versions